babel.config.js

```diff
@@ -1,4 +1,8 @@
 // Babel configuration currently only used by Jest.
 module.exports = {
-  presets: ['@babel/preset-env', '@babel/preset-react'],
+  presets: [
+    '@babel/preset-env',
+    ['@babel/preset-typescript', { allowDeclareFields: true }],
```

Comment:
why allowDeclareFields=true?

Reply:
We use the TS `declare` keyword in class components to use React context